exclude_list.yaml

```diff
@@ -0,0 +1,11 @@
+# List of Python packages to exclude from automatically assembled requirements
+#"From assembled <main requirements> exclude <package_name> with <version> on <platform>".
+
+# exclude_list template
```

Comment:
How can we specify Python version restrictions, e.g. package cannot be build with specific Python versions?

Reply:
Since we are building only for Python 3.8 I think it will basically have no effect to do so (to build or not to build should be sufficient).
But for some wheels which are Python version specific maybe we should build the wheels for other Python versions and specify the version in exclude/include lists will be needed.

Maybe this is the topic for further discussion in person?